elementary/monitor/cli.py

```diff
@@ -115,6 +115,7 @@ def monitor(
         slack_webhook,
         slack_token,
         slack_channel_name,
+		  slack_timezone,
```

Comment:
Don't forget to add the right decorator as well!
```
@click.option(
    '--slack-timezone', '-tz',
    type=str,
    default=None,
    help="The timezone of which all timestamps will be converted to. (default is user local timezone)",
)
```

Reply:
Plus the timezone is part of the monitor content and agnostics to the platform it is shared on.
So I think that a better name is just `timezone` or `alerts_timezone` 🙂

Reply:
changed to `timezone`